src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -213,6 +214,31 @@ function bindEventsAfterAjax() {
     setupFsCopyModal();
 }
 
+function escapeHtml(unsafe) {
+    return unsafe.replace(/&/g, '&amp;').replace(/</g, '&lt;')
+            .replace(/>/g, '&gt;').replace(/"/g, '&quot;')
+            .replace(/'/g, '&#039;');
+}
+
+function initializeCourseName() {
+    $('.course-name-data').each((idx, obj) => {
+        const [value] = [obj.value];
```

Comment:
Instead of doing this, consider using [jQuery `data-*` attribute](https://api.jquery.com/data/) instead.